### PR TITLE
Add Wishlist and cart management page

### PR DESCRIPTION
**Pull request by AnkurChunekar**
cart pr ss

- Fixed card components 
- add wishlist page
- add cart page 
- Made all pages responsive
- add links to anchor tags

### CODE REVIEW

File: pages/cart-management/cart-management.css [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+/* order box ends */
+
+/* media query starts */
+
+@media screen and (max-width: 1024px) {
+
+  html {
+    font-size: 50%;
+  }
```
Comment (AtulPant2704) [inline pages/cart-management/cart-management.css:102]: Just a small suggestion here you can have next line blank, as you are leaving next line blank after every curly braces end so you should continue your that approach and have same indentation everywhere.

File: pages/wishlist/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+              <footer class="card-actions">
+                <button class="btn btn-primary card-btn-1">Move to Cart</button>
+                <button class="btn btn-secondary card-btn-2">Add to Cart</button>
+              </footer>
+       
+            </section>
+          </div>
+
+
+        <!-- card here-->
```
Comment (gadgetfather) [inline pages/wishlist/wishlist.html]: you can write ```<!-- x card here x-->``` so it becomes easier to understand that card part has ended here.

**Review by gadgetfather — APPROVED** [review]
Good work 👍

**Review by AtulPant2704 — APPROVED** [review]
Cards are looking really good.

File: pages/wishlist/wishlist.html (hunk 2 above)
Comment (AnkurChunekar) [reply pages/wishlist/wishlist.html]: Thanks a lot, for your suggestion, I'll make the necessary changes asap
!

File: pages/cart-management/cart-management.css (hunk 1 above)
Comment (AnkurChunekar) [reply pages/cart-management/cart-management.css:102]: Thanks atul for pointing it out, missed it!, i'll do it asap.

**Pull request merged**